src/Tasks/ResolveAssemblyReferences/Server/RarController.cs

```diff
@@ -130,9 +126,31 @@ public async Task<int> StartAsync(CancellationToken cancellationToken = default)
             return 0;
         }
 
-        private async Task HandleClientAsync(Stream serverStream, CancellationToken cancellationToken = default)
+        private async Task<Stream?> ConnectAsync(CancellationToken cancellationToken = default)
+        {
+            Stream serverStream = GetStream(_pipeName);
+
+            if (!(serverStream is NamedPipeServerStream pipeServerStream))
+            {
+                return serverStream;
```

Comment:
Can you comment this please? I don't understand the reasoning for the early return.

Reply:
This is for testing purposes. In testing scenarios I want to pass streams which I have created. And normal streams doesn't support rest of the method, so for this reason I am making early exit.